HIGHLIGHTS.md

```diff
@@ -5,5 +5,7 @@
 What highlights can you come up with to share with the class?
 
 What do you really like? What can you find for fun facts related to any of these topics online?
+- The most fun part for Erick is using npm packages!
+- API integrations such as Auth0 and google apps.
```

Comment:
Is Auth0 a security feature?

Reply:
Auth0 is a way to let another service be your authentication gateway.

Think about Doordash, how you can login via Google and Facebook. They use Auth0 authentication to authorize your login without Doordash ever getting your Facebook password.